test/ca/mcgill/jetuml/layouttests/AbstractTestDiagramLayout.java

```diff
@@ -108,9 +115,66 @@ protected Edge edgeByMiddleLabel(String pLabel)
 				.collect(Collectors.toUnmodifiableList());
 	}
 	
+	protected static int getStaticIntFieldValue(Class<?> pClass, String pFieldName) throws ReflectiveOperationException

```

Comment:
It's better to absorb the exception. We can use a try-catch with a `assert false; fail();` in the catch block. Because this should basically not happen unless there's a major bug in the test. This will result in a general decluttering of the test code.